src/codewhisperer/util/userGroupUtil.ts

```diff
@@ -51,10 +51,7 @@ export class CodeWhispererUserGroupSettings {
     }
 
     private guessUserGroup(): UserGroup {
-        const randomNum = Math.random()
-        const result = randomNum <= 1 / 2 ? UserGroup.Control : UserGroup.CrossFile
-
-        return result
```

Comment:
This change conflicts with https://github.com/aws/aws-toolkit-vscode/pull/3651/files#diff-46248c6fb980971704762e9444db5d26ea68a7326bd12ccbf92fea0ce755034eR55

Reply:
yea this one will be merged first, the UTG one should be the final look

Reply:
Just seperate these 2 out in 2 PRs
1. remove cross file A/B
2. add usg A/B